gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Inject, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { catchError, map, switchMap, takeUntil, tap } from 'rxjs/operators';
+import { EMPTY, merge, Subject } from 'rxjs';
+import { StateParams } from '@uirouter/angularjs';
+
+import { UIRouterStateParams } from '../../../../../ajs-upgraded-providers';
+import { ApiV2Service } from '../../../../../services-ngx/api-v2.service';
+import { Analytics, ApiV4, SamplingTypeEnum } from '../../../../../entities/management-api-v2';
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+
+@Component({
+  selector: 'api-runtime-logs-message-settings',
+  template: require('./api-runtime-logs-message-settings.component.html'),
+  styles: [require('./api-runtime-logs-message-settings.component.scss')],
+})
+export class ApiRuntimeLogsMessageSettingsComponent implements OnInit, OnDestroy {
+  @Input() public api: ApiV4;
+  form: FormGroup;
+  samplingType: SamplingTypeEnum;
+  hasLoggingModeEnabled = false;
```

Comment:
What about changing this to `loggingModeDisabled` because you use it almost always as `!hasLoggingModeEnabled` in the html and component?